.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- Just like NASA going to the moon, it's always good to have a checklist when
+creating changes.
+The following items are listed to help you create a great Pull Request: -->
+## Checklist
+- [ ] All new and existing **tests are passing**
```

Comment:
Why does this need a checklist - doesn't running the github actions cover this?

Reply:
- Ideally, GH Actions should not be the first instance PR authors come across the tests. Getting the tests to pass by pushing to (re-)trigger CI runs would be very tedious - we'd want authors to have the tests running locally so they can understand and fix errors faster.
  I'd also wager a lot of authors most likely won't even register the tiny red/green check marks.
- Spelling the need for working tests out explicitly (even if it may be a repetition of the "contributing" section of the docs) is a good reminder for authors, even already when thinking about submitting drafts

Reply:
Using some sort of CI system (all of which are reported with check masks) is common for everything on GitHub. The more text you include here, the less likely people are to actually read all of it.

Reply:
I've reduced the amount of text and bullet points even further.

I am convinced and have seen evidence of this type of checklist being useful - and though I hate this type of argument, it's also something other successful projects do.